deliveries/Archisinal-milestone_1.md

```diff
@@ -0,0 +1,41 @@
+# Milestone Delivery :mailbox:
+
+> ⚡ Only the GitHub account that submitted the application is allowed to submit milestones. 
```

Comment:
Oh yeah, all the lines starting with `>` can be removed

Reply:
👍